controllers/horizon_controller.go

```diff
@@ -785,6 +785,7 @@ func (r *HorizonReconciler) generateServiceConfigMaps(
 		"ServerName":         fmt.Sprintf("%s.%s.svc", horizon.ServiceName, instance.Namespace),
 		"Port":               horizon.HorizonPort,
 		"TLS":                false,
+		"isHTTPS":            url.Scheme == "https",
```

Comment:
nit: could make it clearer that this refers to the public endpoint

Reply:
Good point. Updated.